src/ComplexityVisitor.js

```diff
@@ -26,18 +64,33 @@ export default class ComplexityVisitor {
     this.costFactor = 1;
 
     this.rootCalculator = new CostCalculator();
-    this.fragmentCalculators = Object.create(null);
 
     this.Field = {
       enter: this.enterField,
       leave: this.leaveField,
     };
 
-    this.FragmentSpread = this.enterFragmentSpread;
+    this.FragmentDefinition = () => {
+      // don't visit any further we will include these at the spread location
+      return false;
+    };
```

Comment:
logic for skipping and editing assumed to be correct as noted here: https://github.com/graphql/graphql-js/blob/master/src/language/visitor.js#L140

graphql-express and others do not use the edited AST so this should be safe